axelor-base/src/main/resources/views/Partner.xml

```diff
@@ -942,7 +942,11 @@
     <action-validate name="action-partner-validate-partner-stock-settings-list">
     	<alert message="Partner has duplicate stock settings for same company" if="partnerStockSettingsList.collect{it->it.company}.unique().size() &lt; partnerStockSettingsList.size"/>
     </action-validate>
-
+   	 
+   	<action-attrs name="action-partner-attrs-user-group" if-module="axelor-crm">
+		<attribute name="domain" for="user" expr="eval: &quot;self.group IN (${__config__.app.getApp('crm').groupsAssignable?.collect{it.id}.join(',')})&quot;" if="__config__.app.getApp('crm').assignableUser == 2"/>
```

Comment:
It will throw error when there is no group selected on 'groupsAssignable' , handle this case too.